scim/Package.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is the requirement to have this? if we don't have anything to add to here, can we remove this?